src/components/reusable/nav/mod.rs

```diff
@@ -52,6 +53,28 @@ pub fn Nav(cx: Scope<Props>) -> Element {
             loop {
                 let list = multipass.list_incoming_request().unwrap_or_default();
                 if list.len() != *reqCount.get() {
+                    // If list is updated, we update the request count
+                    if list.len() > *reqCount.get() {
+                        // We display a notification if the list length is increased (incoming request appended).
+                        // We do not display a notification if the list length is decreased (incoming request is rejected).
+
+                        // TODO: Display sender name of incoming friend request
+                        let most_recent_friend_request = list.last().unwrap().from(); // This returns a FriendRequest struct
+                                                                                      // TODO: Get username from DID: get_username_from_did(most_recent_friend_request, &mp);
+                                                                                      // Where &mp is a lifetime. In other file it is usually: let mp = &cx.props.account.clone();
+                                                                                      // Lifetime issue:
+                                                                                      // lifetime may not live long enough
+                                                                                      // returning this value requires that `'1` must outlive `'static`
+
+                        let _username =  // does this address the TODO? 
```

Comment:
i think this addresses one of the TODOs

Reply:
Was the username going to be used in some way? If not, it could get removed

Reply:
I communicated it to Sheldon last night, the description notification feature IIRC is only for macOS so it's not that important. I've removed it.